src/middlewared/middlewared/plugins/disk_/sync.py

```diff
@@ -86,7 +83,6 @@ async def sync_all(self, job):
                 self.logger.warning('Starting disk.sync_all when devd is not connected yet')
 
         sys_disks = await self.middleware.call('device.get_disks')
-        self.logger.info('Found disks: %r', sys_disks)
```

Comment:
@themylogin  do you remember why we put it there? Should we log at least the names?

Reply:
Yes, we have issues when syncing disks not resolved yet, this actually helped me once. @yocalebo can we please leave this line and log only `name`, `ident`, `serial` and `lunid`?

Reply:
Yeah I can do that. But `ident` and `serial` are the same right?